local_output/Tiltfile

```diff
@@ -0,0 +1,2 @@
+def local_output(command):
```

Comment:
Suggest also adding another method to return the most recent exit code. This would allow something like `dirExists = local_exitCode('test -d ./some/path') == 0`

Reply:
or possibly returning a tuple:
```
result, exitCode = local_output(command)
```

Reply:
or possibly returning a tuple:
```
result, exitCode = local_output(command)
```